src/scss/_error.scss

```diff
@@ -0,0 +1,12 @@
+.error {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
```

Comment:
Do you mean the input dropdown in the authoring page? This code/pattern probably isn't relevant to that anyway, because those errors would be occurring in the absence of a storyline instance.